haystack_experimental/components/tools/openapi/_openapi.py

```diff
@@ -131,7 +131,7 @@ def __init__(
         request_sender: Optional[Callable[[Dict[str, Any]], Dict[str, Any]]] = None,
         llm_provider: LLMProvider = LLMProvider.OPENAI,
         operations_filter: Optional[Callable[[Dict[str, Any]], bool]] = None,
-    ):  # noqa: PLR0913 # pylint: disable=too-many-positional-arguments
+    ):  # noqa: PLR0913
```

Comment:
This and the following change revert #101.

They seem unrelated to this PR, but probably stem from the fact that we were not using `actions/setup-python@v5` in the `linting` job, so Python (and `pylint`) versions may have been different than expected. The current version of `pylint` does not recognize this argument: `Unknown option value for '--disable', expected a valid pylint message and got 'too-many-positional-arguments'`.